app/assets/javascripts/main.mjs

```diff
@@ -1,7 +1,15 @@
-import { Frontend } from './govuk-frontend-local';
+import { Button, createAll, ErrorSummary, SkipLink } from 'govuk-frontend'
 
-Frontend.initAll();
+createAll(Button)
 
-export {
-  Frontend
+// Find first error summary module to enhance.
```

Comment:
Should we just use `createAll` to instatiate this module and the `$skipLink`? The guidance seems to say we should only not use it if we intend to handle any errors thrown ourselves, which we don't.

Reply:
yep, correct we can just use that. updated